Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/detectiontracker/DefaultExposureDetectionTracker.kt

```diff
@@ -93,34 +94,16 @@ class DefaultExposureDetectionTracker @Inject constructor(
         }
     }
 
-    override fun finishExposureDetection(identifier: String, result: Result) {
-        Timber.i("finishExposureDetection(token=%s, result=%s)", identifier, result)
```

Comment:
Please re-add a log statement directly at function start, the following methods using `updateSafely` are async, if there is any issue there we'd still like to know that the function was called, but failed somewhere later.

Reply:
added.